http/headers/Content-Encoding.json

```
@@ -4,7 +4,10 @@
       "Content-Encoding": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Reference/Headers/Content-Encoding",
-          "spec_url": "https://httpwg.org/specs/rfc9110.html#field.content-encoding",
+          "spec_url": [
+            "https://www.rfc-editor.org/rfc/rfc8878#name-content-encoding",
+            "https://httpwg.org/specs/rfc9110.html#field.content-encoding"
```

Same as above, although we might want to keep the order.

```suggestion
            "https://httpwg.org/specs/rfc9110.html#field.content-encoding",
            "https://www.rfc-editor.org/rfc/rfc8878#name-content-encoding"
```